README.md

```diff
@@ -0,0 +1,19 @@
+# TYPO3 Trainer Network Distribution
+
+This TYPO3 distribution contains an example TYPO3 installation.
+This installation includes all Extensions and examples used during official Trainings.
+
+## Installation
+
+1. Run `composer install`
+
+2. Run `TYPO3_CONTEXT=Development php -S 127.0.0.1:8080 -t public`
```

Comment:
No blank line between list items, please.

Also, please always use `1.`. MarkDown will automatically increase the number. This reduces the work when lines are switched, added or removed.